### PR TITLE
Fix #10909: Remove deprecated code

### LINKED ISSUE #10909: Remove Code in 15.0.0 Reminder
- [x] Remove Apache Commons FileUpload stuff (https://github.com/primefaces/primefaces/issues/10751)
- [x] Remove InputSwitch (https://github.com/primefaces/primefaces/issues/10874)
- [x] Remove legacy Chart.JS components and all PF models (https://github.com/primefaces/primefaces/issues/10870)
- [x] Remove `@Deprecated` legacy "table" layouts
- [x] Remove `@Deprecated` FileUpload properties
- [x] Remove `@Deprecated` DataExporter code

### PR DESCRIPTION
**Pull request by melloware**
Fix #10909: Remove deprecated code

@Rapster just two i need you to look at after this PR and its `DataExporter` and `FileUpload` some things marked deprecated looked like they were still in use so I didn't touch them.

### CODE REVIEW

File: primefaces/src/main/java/org/primefaces/component/selectmanycheckbox/SelectManyCheckboxRenderer.java [hunk 1]
```diff
@@ -82,9 +82,6 @@ protected void encodeMarkup(FacesContext context, SelectManyCheckbox checkbox) t
             encodeCustomLayout(context, checkbox);
             writer.endElement("span");
         }
-        else if ("grid".equals(layout)) {
-            encodeLegacyTabularLayout(context, checkbox, layout);
```
Comment (jepsar) [inline primefaces/src/main/java/org/primefaces/component/selectmanycheckbox/SelectManyCheckboxRenderer.java:86]: Maybe keep the else if and throw an exception?

File: primefaces/src/main/java/org/primefaces/component/selectoneradio/SelectOneRadioRenderer.java [hunk 2]
```diff
@@ -69,9 +69,6 @@ protected void encodeMarkup(FacesContext context, SelectOneRadio radio) throws I
         if (custom) {
             encodeCustomLayout(context, radio);
         }
-        else if ("grid".equals(layout)) {
-            encodeLegacyTabularLayout(context, radio, layout);
```
Comment (jepsar) [inline primefaces/src/main/java/org/primefaces/component/selectoneradio/SelectOneRadioRenderer.java:73]: Throw?

File: primefaces/src/main/java/org/primefaces/component/selectmanycheckbox/SelectManyCheckboxRenderer.java (hunk 1 above)
Comment (melloware) [reply primefaces/src/main/java/org/primefaces/component/selectmanycheckbox/SelectManyCheckboxRenderer.java:86]: good idea

**Comment by tandraschko** [comment]
are we sure we wanna remove tabular support? i can remember a discussion here and that i fixed something regarding :D

**Comment by melloware** [comment]
We kept it in PanelGrid a its SO popular there and we removed the deprecation because so many people use PanelGrid with tabular :)

**Pull request merged**